examples/webgl_reverse_depth_buffer.html

```diff
@@ -232,7 +232,7 @@
 				logarithmicComposer.addPass( new OutputPass() );
 
 				const reverseContainer = document.getElementById( 'container_reverse' );
-				reverseRenderer = new THREE.WebGLRenderer( { reverseDepthBuffer: true } );
+				reverseRenderer = new THREE.WebGLRenderer( { reversedDepthBuffer: true } );
```

Comment:
Given that the reverse depth buffer was relatively new and somewhat experimental, I think we can quit the deprecation warning. A note in the migration guide about the renaming should be sufficient.